docs/howto/openstack/octavia/lbaas-tcp.md

```diff
@@ -0,0 +1,742 @@
+Yello from srv-lbaas-1!
+```
+
+It looks like we talked to the first test server. If we run the
+exact same `wget` command for a second time, since the load balancer
+distributes client connection requests in a round-robin fashion,
+we expect to talk to the second server:
+
+```bash
+wget -q 198.51.100.129:61234 -O -
+```
+
+```plain
+Yello from srv-lbaas-2!
+```
+
+During our testing, executing the `wget` command repeatedly, we were
+getting those two hostnames one after the other --- and that was
+proof our TCP load balancer was working as expected. But there is
+one more expectation of any load balancer; the ability to skip
+backend hosts when they are inaccessible.
+
+To test our load balancer in this new scenario, let us first use
+`wget` to connect to port 61234 and jot down the backend server
+that will respond:
+
+```bash
+wget -q 198.51.100.129:61234 -O -
+```
+
+```plain
+Yello from srv-lbaas-2!
+```
+
+We see that `srv-lbaas-2` responded. That means the next time we
+try to connect, `srv-lbaas-1` will respond. But if we SSH into
+`srv-lbaas-1` and terminate `ncat`, then after connecting with
+`wget` we will get a response from `srv-lbaas-2` --- again. This,
+at least, is our expectation. So without further ado, we SSH into
+`srv-lbaas-1`, we terminate `ncat`, we log out, and from our local
+terminal, we type:
+
+```bash
+wget -q 198.51.100.129:61234 -O -
+```
+
+```plain
+Yello from srv-lbaas-2!
+```
+
+Exactly as we expected, `srv-lbaas-2` has responded again, proving
+our load balancer behaves correctly. As an exercise, you might want
+to shutdown your `srv-lbaas-1` and/or re-enable `ncat`, and check
+the behavior of the load balancer. There should be no surprises.
+
+## Adding a health monitor
+
+You can add a health monitor to the pool of your load balancer,
+so whenever ---and for whatever reason--- one or more of the pool
+members are inaccessible, the load balancer will know and won't
+even bother trying to redirect client requests to them. Of course,
+whenever an inaccessible pool member gets accessible again, the
+load balancer will take notice and start treating that member as
+a fully functional server.
+
+=== "{{gui}}"
+    In the detailed view of your load balancer, pull up the
+    _Pools_ tab. In the _Health Monitor_ column there is a "0",
+    for the pool has no health monitor yet.
+
+    ![There is no health monitor](assets/shot-23.png)
+    
+    To add a health monitor, click the notepad-and-pen icon.
+    A pane titled _Modify Pool_ slides over. Scroll down if
+    you have to, and stop when the _Health Monitor_ section is
+    fully visible. You will see a message saying, "No health
+    monitor created", so click the green _Create a Healthmonitor_
+    button to create one.
+    
+    ![Create new health monitor](assets/shot-24.png)
+    
+    Pick a name for the new health monitor, and set its type to
+    PING. That way, ICMP echo requests will be used to determine
+    whether pool members are accessible.
```

Comment:
Let's please not do that; that's another bad practice we'd prefer users not to do. Whether a _host_ is pingable really has no bearing on whether the _service_ is accessible. If the host is up but the service has been stopped, we want that host to be dropped from the pool, and when we use a `PING` health monitor we don't. So let's use a `TCP` one instead.

Reply:
I see --- and I will update accordingly!